pkg/eventprocessor/impl/executor/impl/txnscope_createtable.go

```diff
@@ -0,0 +1,81 @@
+package impl
```

Comment:
Note how I separate the business logic of each event type in `txnscope_xxxxx.go`.
This allows finding fast all the logic corresponding to particular event execution. 

I did the same with tests; they live in `txnscope_xxxx_test.go`, so we can also find the tests related to an event type quite fast.